src/helpers/responsiveness.ts

```diff
@@ -2,8 +2,7 @@ import { css } from 'styled-components';
 
 export type TDeviceSizes = 'tablet' | 'phone' | 'desktop';
 
-// TODO: Consider the breakpoints defined by google's material design https://material.io/guidelines/layout/responsive-ui.html#responsive-ui-breakpoints
-const sizes = {
+export const breakpoints = {
```

Comment:
I think a better typing would be
```suggestion
export const breakpoints = {
  desktop: 1280,
  phone: 600,
  tablet: 720,
} as const;

export type TDeviceSizes = keyof typeof breakpoints;

type TMedia = Record<TDeviceSizes, ReturnType<typeof interpolate>>;
  
const maxMedia: TMedia = {
  phone: interpolate('phone', 'max'),
  tablet: interpolate('tablet', 'max'),
  desktop: interpolate('desktop', 'max'),
};

const minMedia: TMedia = {
  phone: interpolate('phone', 'min'),
  tablet: interpolate('tablet', 'min'),
  desktop: interpolate('desktop', 'min'),
};
```